third_party/amd/lib/TritonAMDGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMFMA.cpp

```diff
@@ -320,7 +320,7 @@ Value convertLayout(int opIdx, ConversionPatternRewriter &rewriter,
           spatialWarpId, lane, warpsPerBlockNonK, numOfElems, numReps, smemObj,
           sharedLayout, nDim, mfmaInstrK);
     }
-    smemBase = AMD::computeBasePtr(rewriter, loc, smemObj);
+    smemBase = smemObj.getBaseBeforeSlice(order[0], loc, rewriter);
```

Comment:
@zhanglx13 I think this fix was from you originally. I saw that `computeBasePtr` is also used in other places. Should we separate this change out and fix everywhere? Or are those fine? Could you explain more here?

Reply:
This is the only place that we use `computeBasePtr` (the other place is for wmma) and it only matters when we take a slice of LDS.
I need to dig into my notes to learn about why I changed it.